pkg/apis/cnsoperator/config/cnsregistervolume_crd.yaml

```diff
@@ -55,9 +55,9 @@ spec:
           type: object
           description: CnsRegisterVolumeStatus defines the observed state of CnsRegisterVolume
           properties:
-            success:
```

Comment:
Any idea how we missed this? What is the upgrade impact?

Reply:
May be because, we have defined the `CnsRegisterVolumeStatus` as:
```
type CnsRegisterVolumeStatus struct {
	// Indicates the volume is successfully registered.
	// This field must only be set by the entity completing the register
	// operation, i.e. the CNS Operator.
	Registered bool `json:"registered"`

	// The last error encountered during import operation, if any.
	// This field must only be set by the entity completing the import
	// operation, i.e. the CNS Operator.
	Error string `json:"error,omitempty"`
}
```
as mentioned here https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/apis/cnsoperator/cnsregistervolume/v1alpha1/cnsregistervolume_types.go#L54-L63 

In all the places we are referring to the field name as `Registered`, even in our e2e tests.

Reply:
> What is the upgrade impact?

Actually None!. I have added the observations here:
https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/554#issuecomment-749336100